nightly.sh

```diff
@@ -15,15 +15,21 @@ REPO_DIR="${HTTPROOT}/$(repo_path ${REPO})"
 main() {
     export NIGHTLY=1 STATIC_TESTS=0 SAVE_JOB_RESULTS=1
 
+    echo -n "--- flushing redis test result cache: "
+    redis-cli flushall
+
     for branch in $BRANCHES; do
         local commit="$(gethead $REPO $branch)"
-        local output_dir="${REPO_DIR}/$branch/${commit}"
+        local output_dir_commit="${REPO_DIR}/$branch/${commit}"
+        local output_dir="${output_dir_commit}.$(date +'%Y%m%d%H%M')"
+        local latest_link="$(dirname "$output_dir")/latest"
```

Comment:
Duplicates [the line below](https://github.com/RIOT-OS/murdock-scripts/pull/23/files#diff-a662779f8c0f262d3810ffb1af1362d4R29).

Reply:
yup, thx!

Reply:
I'll sqash this right in so we don't accidentally merge the fixup :)